docs/ota-client-guide/modules/ROOT/pages/troubleshooting.adoc

```diff
@@ -44,7 +44,7 @@ GnuTLS currently requires a kernel >= 3.17 to build. If you're using an older ke
  #  define getrandom(dst,s,flags) syscall(SYS_getrandom, (void*)dst, (size_t)s, (unsigned int)flags)
 ----
 
-This is an upstream problem; until it's fixed in upstream poky, you can link:../files/gnutls-fix.patch[download this patch] to your `poky` directory and apply it to fix the issue.
+This is an upstream problem; until it's fixed in upstream poky, you can link:../../files/gnutls-fix.patch[download this patch] to your `poky` directory and apply it to fix the issue.
```

Comment:
Should be an absolute link.

Reply:
> Should be an absolute link.

The link was relative on the Jekyll-based site too. It was hosted on the docs server for some reason. I.e. there was literally a directory called "files" with this one patch file in it (so I replicated it on the Antora-based site). Any idea where we should put this instead?

Reply:
Oops, didn't realize that. I just looked, and this has been patched upstream in every poky version. We can remove this section altogether.

Reply:
In general, in Antora, we should never be using position-relative links (i.e. `../whatever`). It should almost always be an xref. If an xref doesn't work because it's linking to another catalog or module, we should either use an `ifdef::env-github[]` to make it work in both locations, or just leave the unresolvable xref if the link isn't super important. (The github renderer has a pretty reasonable failure mode there: it will just remove the link, but leave the text.)